examples/simple/main.tf

```diff
@@ -92,17 +92,16 @@ variable "tags" {
 variable "materialize_instances" {
   description = "Configuration for Materialize instances"
   type = list(object({
-    name                 = string
-    namespace            = optional(string)
-    database_name        = string
-    environmentd_version = optional(string, "v0.131.1")
-    cpu_request          = optional(string, "1")
-    memory_request       = optional(string, "1Gi")
-    memory_limit         = optional(string, "1Gi")
-    create_database      = optional(bool, true)
-    in_place_rollout     = optional(bool, false)
-    request_rollout      = optional(string)
-    force_rollout        = optional(string)
+    name             = string
```

Comment:
So ... when I ran this yesterday, I did have to specify my subscription id on line 21 (I had az login and selected my subscription)  https://github.com/MaterializeInc/terraform-azurerm-materialize/pull/4/files#diff-b887d6095ae6f83dd7594063ca62cb3545f6c8089f3bb693d607566e73c4eeadR21 --